tests/test_packaging.py

```diff
@@ -184,8 +184,8 @@ def test_polish_and_ratchet_pins_and_upper_bounds():
     )
 
     # Polish and ratchet this number down as low as possible
-    assert len(sorted_packages_with_pins_or_upper_bounds) == 81
-    assert sorted_packages_with_pins_or_upper_bounds == [
+    assert len(sorted_packages_with_pins_or_upper_bounds) == 85
+    assert set(sorted_packages_with_pins_or_upper_bounds) == {
```

Comment:
@anthonyburdi might have something to say about this, but we can deal with that later.

Reply:
I think pinning pandas is fine as a temporary measure. If it is only that specific version you can use the `!=` instead of `<`, the benefit being we would not have to remember to remove the upper bound after fixing the underlying issue if it is only present in one specific version.

The list -> set change I think is fine. It was only a list to make it a little more human readable.

If we move to poetry I think this test goes away.